G/GStreamer/build_tarballs.jl

```diff
@@ -42,13 +49,15 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="Glib_jll", uuid="7746bdde-850d-59dc-9ae8-88ece973131d"), v"2.59.0"; compat="2.59.0")
+    # Need a host gettext for msgfmt
+    HostBuildDependency("Gettext_jll")
+    Dependency(PackageSpec(name="Glib_jll", uuid="7746bdde-850d-59dc-9ae8-88ece973131d"); compat="2.68.1")
     Dependency(PackageSpec(name="LibUnwind_jll", uuid="745a5e78-f969-53e9-954f-d19f2f74f4e3"))
     Dependency(PackageSpec(name="Elfutils_jll", uuid="ab5a07f8-06af-567f-a878-e8bb879eba5a"))
```

Comment:
Elfutils is only for Linux. I'm not even sure it's actually used?

Reply:
I don't know either. I just kept it as it was.

Reply:
```suggestion
    Dependency(PackageSpec(name="Elfutils_jll", uuid="ab5a07f8-06af-567f-a878-e8bb879eba5a"); platforms=filter(Sys.islinux, platforms))
```